base/src/commonMain/kotlin/flow/Flowable.kt

```diff
@@ -45,11 +57,6 @@ interface Flowable<T> {
     /**
      * Applies [set] in a blocking coroutine
      */
-    fun setBlocking(value: T)
+    fun setBlocking(value: T, context: CoroutineContext = EmptyCoroutineContext) = runBlocking { set(value) }
```

Comment:
I know its deprecated but it looks like context is never used (not in this default impl nor in the implementations of this interface)

Reply:
indeed it's supposed to be passed to the runblocking on the right side, but it was before I deprecated Flowable, will remove.

Reply:
removed in architecture branch.